App dev crash - Photos

When choosing from the "Photos" menu, the app crashes.

```
************ CAUSE OF ERROR ************

java.lang.IllegalStateException: View 'ListItemLayout' with ID 2131296262 for field 'itemLayout' was of the wrong type. See cause for more info.
	at butterknife.internal.Utils.castView(Utils.java:105)
	at butterknife.internal.Utils.findRequiredViewAsType(Utils.java:97)
	at com.owncloud.android.ui.adapter.OCFileListAdapter$OCFileListGridImageViewHolder_ViewBinding.<init>(OCFileListAdapter$OCFileListGridImageViewHolder_ViewBinding.java:30)
	at java.lang.reflect.Constructor.newInstance0(Native Method)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:343)
	at butterknife.ButterKnife.bind(ButterKnife.java:170)
	at com.owncloud.android.ui.adapter.OCFileListAdapter$OCFileListGridImageViewHolder.<init>(OCFileListAdapter.java:1308)
	at com.owncloud.android.ui.adapter.OCFileListAdapter$OCFileListGridImageViewHolder.<init>(OCFileListAdapter.java:1281)
	at com.owncloud.android.ui.adapter.OCFileListAdapter.onCreateViewHolder(OCFileListAdapter.java:314)
	at androidx.recyclerview.widget.RecyclerView$Adapter.createViewHolder(RecyclerView.java:7078)
	at androidx.recyclerview.widget.RecyclerView$Recycler.tryGetViewHolderForPositionByDeadline(RecyclerView.java:6235)
	at androidx.recyclerview.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:6118)
	at androidx.recyclerview.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:6114)
	at androidx.recyclerview.widget.LinearLayoutManager$LayoutState.next(LinearLayoutManager.java:2303)
	at androidx.recyclerview.widget.GridLayoutManager.layoutChunk(GridLayoutManager.java:561)
	at androidx.recyclerview.widget.LinearLayoutManager.fill(LinearLayoutManager.java:1587)
	at androidx.recyclerview.widget.LinearLayoutManager.onLayoutChildren(LinearLayoutManager.java:665)
	at androidx.recyclerview.widget.GridLayoutManager.onLayoutChildren(GridLayoutManager.java:170)
	at androidx.recyclerview.widget.RecyclerView.dispatchLayoutStep2(RecyclerView.java:4134)
	at androidx.recyclerview.widget.RecyclerView.dispatchLayout(RecyclerView.java:3851)
	at androidx.recyclerview.widget.RecyclerView.onLayout(RecyclerView.java:4404)
	at android.view.View.layout(View.java:22114)
	at android.view.ViewGroup.layout(ViewGroup.java:6335)
	at androidx.swiperefreshlayout.widget.SwipeRefreshLayout.onLayout(SwipeRefreshLayout.java:625)
	at android.view.View.layout(View.java:22114)
	at android.view.ViewGroup.layout(ViewGroup.java:6335)
	at com.google.android.material.appbar.HeaderScrollingViewBehavior.layoutChild(HeaderScrollingViewBehavior.java:148)
	at com.google.android.material.appbar.ViewOffsetBehavior.onLayoutChild(ViewOffsetBehavior.java:43)
	at com.google.android.material.appbar.AppBarLayout$ScrollingViewBehavior.onLayoutChild(AppBarLayout.java:1892)
	at androidx.coordinatorlayout.widget.CoordinatorLayout.onLayout(CoordinatorLayout.java:918)
	at android.view.View.layout(View.java:22114)
	at android.view.ViewGroup.layout(ViewGroup.java:6335)
	at android.widget.RelativeLayout.onLayout(RelativeLayout.java:1103)
	at android.view.View.layout(View.java:22114)
	at android.view.ViewGroup.layout(ViewGroup.java:6335)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:334)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
	at android.view.View.layout(View.java:22114)
	at android.view.ViewGroup.layout(ViewGroup.java:6335)
	at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1839)
	at android.widget.LinearLayout.layoutHorizontal(LinearLayout.java:1828)
	at android.widget.LinearLayout.onLayout(LinearLayout.java:1594)
	at android.view.View.layout(View.java:22114)
	at android.view.ViewGroup.layout(ViewGroup.java:6335)
	at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1839)
	at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1683)
	at android.widget.LinearLayout.onLayout(LinearLayout.java:1592)
	at android.view.View.layout(View.java:22114)
	at android.view.ViewGroup.layout(ViewGroup.java:6335)
	at androidx.drawerlayout.widget.DrawerLayout.onLayout(DrawerLayout.java:1231)
	at android.view.View.layout(View.java:22114)
	at android.view.ViewGroup.layout(ViewGroup.java:6335)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:334)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
	at android.view.View.layout(View.java:22114)
	at android.view.ViewGroup.layout(ViewGroup.java:6335)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:334)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
	at android.view.View.layout(View.java:22114)
	at android.view.ViewGroup.layout(ViewGroup.java:6335)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:334)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
	at android.view.View.layout(View.java:22114)
	at android.view.ViewGroup.layout(ViewGroup.java:6335)
	at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1839)
	at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1683)
	at android.widget.LinearLayout.onLayout(LinearLayout.java:1592)
	at android.view.View.layout(View.java:22114)
	at android.view.ViewGroup.layout(ViewGroup.java:6335)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:334)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:270)
	at com.android.internal.policy.DecorView.onLayout(DecorView.java:937)
	at android.view.View.layout(View.java:22114)
	at android.view.ViewGroup.layout(ViewGroup.java:6335)
	at android.view.ViewRootImpl.performLayout(ViewRootImpl.java:3285)
	at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2785)
	at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1857)
	at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:8089)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1057)
	at android.view.Choreographer.doCallbacks(Choreographer.java:875)
	at android.view.Choreographer.doFrame(Choreographer.java:776)
	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:1042)
	at android.os.Handler.handleCallback(Handler.java:888)
	at android.os.Handler.dispatchMessage(Handler.java:100)
	at android.os.Looper.loop(Looper.java:213)
	at android.app.ActivityThread.main(ActivityThread.java:8178)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:513)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1101)
Caused by: java.lang.ClassCastException: Cannot cast android.widget.FrameLayout to android.widget.LinearLayout
	at java.lang.Class.cast(Class.java:2446)
	at butterknife.internal.Utils.castView(Utils.java:102)
	... 88 more

************ APP INFORMATION ************
ID: com.nextcloud.android.beta
Version: 20200625
Build flavor: versionDev
```

### Environment data
Android version: 10

Nextcloud app version: Dev 20200625

Nextcloud server version: 19.0.0

--- a/src/main/java/com/owncloud/android/ui/adapter/OCFileListAdapter.java
+++ b/src/main/java/com/owncloud/android/ui/adapter/OCFileListAdapter.java
@@ -43,7 +43,6 @@ import android.view.WindowManager;
 import android.widget.Filter;
 import android.widget.FrameLayout;
 import android.widget.ImageView;
-import android.widget.LinearLayout;
 import android.widget.ProgressBar;
 import android.widget.RelativeLayout;
 import android.widget.TextView;
@@ -1298,7 +1297,7 @@ public class OCFileListAdapter extends RecyclerView.Adapter<RecyclerView.ViewHol
         public ImageView checkbox;
 
         @BindView(R.id.ListItemLayout)
-        public LinearLayout itemLayout;
+        public View itemLayout;
 
         @BindView(R.id.unreadComments)
         public ImageView unreadComments;

